satpy/etc/areas.yaml

```diff
@@ -2031,8 +1189,9 @@ moll:
     lower_left_xy: [-18040095.696147293, -9020047.848073646]
     upper_right_xy: [18040095.696147293, 9020047.848073646]
 
-robinson:
-  description: robinson
+greenland_robin_1km:
+  old_name: robinson
+  description: Greenland in Robinson projection
```

Comment:
I would say that if the Robinson projection contained only Greenland and not the whole world, this counts as a bug...